Sample Data/ASIM/CommonSecurityLog_Schema.csv

```diff
@@ -0,0 +1,161 @@
+Protocol,67,"System.String",string
+Reason,68,"System.String",string
+RequestURL,69,"System.String",string
+RequestClientApplication,70,"System.String",string
+RequestContext,71,"System.String",string
+RequestCookies,72,"System.String",string
+RequestMethod,73,"System.String",string
+ReceiptTime,74,"System.String",string
+SourceHostName,75,"System.String",string
+SourceMACAddress,76,"System.String",string
+SourceNTDomain,77,"System.String",string
+SourceDnsDomain,78,"System.String",string
+SourceServiceName,79,"System.String",string
+SourceTranslatedAddress,80,"System.String",string
+SourceTranslatedPort,81,"System.Int32",int
+SourceProcessId,82,"System.Int32",int
+SourceUserPrivileges,83,"System.String",string
+SourceProcessName,84,"System.String",string
+SourcePort,85,"System.Int32",int
+SourceIP,86,"System.String",string
+StartTime,87,"System.DateTime",datetime
+SourceUserID,88,"System.String",string
+SourceUserName,89,"System.String",string
+EventType,90,"System.Int32",int
+DeviceEventCategory,91,"System.String",string
+DeviceCustomIPv6Address1,92,"System.String",string
+DeviceCustomIPv6Address1Label,93,"System.String",string
+DeviceCustomIPv6Address2,94,"System.String",string
+DeviceCustomIPv6Address2Label,95,"System.String",string
+DeviceCustomIPv6Address3,96,"System.String",string
+DeviceCustomIPv6Address3Label,97,"System.String",string
+DeviceCustomIPv6Address4,98,"System.String",string
+DeviceCustomIPv6Address4Label,99,"System.String",string
+DeviceCustomFloatingPoint1,100,"System.Double",real
+DeviceCustomFloatingPoint1Label,101,"System.String",string
+DeviceCustomFloatingPoint2,102,"System.Double",real
+DeviceCustomFloatingPoint2Label,103,"System.String",string
+DeviceCustomFloatingPoint3,104,"System.Double",real
+DeviceCustomFloatingPoint3Label,105,"System.String",string
+DeviceCustomFloatingPoint4,106,"System.Double",real
+DeviceCustomFloatingPoint4Label,107,"System.String",string
+DeviceCustomNumber1,108,"System.Int32",int
+FieldDeviceCustomNumber1,109,"System.Int64",long
+DeviceCustomNumber1Label,110,"System.String",string
+DeviceCustomNumber2,111,"System.Int32",int
+FieldDeviceCustomNumber2,112,"System.Int64",long
+DeviceCustomNumber2Label,113,"System.String",string
+DeviceCustomNumber3,114,"System.Int32",int
+FieldDeviceCustomNumber3,115,"System.Int64",long
+DeviceCustomNumber3Label,116,"System.String",string
+DeviceCustomString1,117,"System.String",string
+DeviceCustomString1Label,118,"System.String",string
+DeviceCustomString2,119,"System.String",string
+DeviceCustomString2Label,120,"System.String",string
+DeviceCustomString3,121,"System.String",string
+DeviceCustomString3Label,122,"System.String",string
+DeviceCustomString4,123,"System.String",string
+DeviceCustomString4Label,124,"System.String",string
+DeviceCustomString5,125,"System.String",string
+DeviceCustomString5Label,126,"System.String",string
+DeviceCustomString6,127,"System.String",string
+DeviceCustomString6Label,128,"System.String",string
```

Comment:
NetworkRuleNumber?

Reply:
DeviceCustomString6 is RuleID in case of IntrusionEvent